readme.md

```diff
@@ -19,6 +19,7 @@ If you have a link or a resource to add, you have a few options:
 Likewise, there are other ways to see how people are implementing IIIF.
 - [Github Topics page](https://github.com/topics/iiif?o=desc&s=updated) - This page will show you the Github repositories that self-identify as IIIF-related, with the most recently updated projects listed first. 
 - [IIIF Community on Zenodo](https://zenodo.org/communities/iiif/) - A Zenodo community for gathering articles and data related to IIIF.
+- [People, projects and institutions active within the IIIF Community on Mastodon](https://julsraemy.ch/mastodiiif/)
```

Comment:
It would be good to get something in there about self-identification, as the GitHub Topics entry has.

Reply:
Thanks. I've just edited my commit.